docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -3,17 +3,37 @@ id: pillars-fabric-components
 title: Fabric Components
 ---
 
-This section contains a high-level introduction to Fabric components. It provides enough context to understand when a Fabric component is needed and how it roughly works.
-It points to the [Renderer](https://reactnative.dev/architecture/fabric-renderer) section of the [Architecture](https://reactnative.dev/architecture/overview) tab for a deep dive into the technical details.
+A Fabric Component is a UI component which is rendered on the screen using the [Fabric Renderer](https://reactnative.dev/architecture/fabric-renderer).
 
-This section must have a warning that it works only with the new architecture enabled. It points to the [migration section](../new-architecture-intro).
+Using Fabric Components instead of Native Components allows us to reap all the [benefit](./why) coming from the **New Architecture**. Specifically, we are able to leverage JSI to efficiently connect the Native UI code and the JavaScript one, and all the other features provided by the new renderer.
+
+A Fabric Component is created starting from a **JS specification**. This, with the help of a bit of [**Codegen**](./pillars-codegen), will create some C++ code, shared among all the RN's platforms. Then, after filling in the missing **native code** with component specific logic, the component can be **integrated** in the app.
+
+The following section will guide you through the creation of a Fabric Component, step-by-step.
+
+:::caution
+Fabric Components only works with the **New Architecture** enabled.
+To migrate to the **New Architecture**, follow the [Migration guide](../new-architecture-intro)
+:::
 
 ## How to Create a Fabric Components
 
-This section is a step-by-step guide to create a Fabric component from scratch. The list of subsections is roughly:
+To create a Fabric Component, we have to follow these steps:
+
+- Define a set of JS specification.
+- Configure the component so that it can be consumed by an app.
```

Comment:
When I specify that is a Fabric Component, I use capital letters (it's a proper name for an Entity in the architecture). When I refer to the entity as a generic item belonging to that class, I used lowercase letter, also not to clutter too much the article.
Does it make sense? Do we prefer to have always a capital C for component?

Reply:
Ok makes sense to me 👍 I'm a big fan of title case, in a sense that if you write `Configure the Component so that` you're implicitely referring to the Fabric Component.

It also makes for more consistent reading for me, but that's really a nit :) up to your preference

Reply:
:+1: I updated the PR. The only lowercase `component` remained is in the first line, which is used together with `UI` so it refers to the class of `UI` components rather than to `Fabric Components`

Reply:
Awesome 👌 resolving here.